src/main/java/com/spotinst/sdkjava/model/SpotinstElastigroupRepo.java

```diff
@@ -1036,4 +1036,94 @@ public RepoGenericResponse<ElastigroupGetInstanceStatusResponse> getInstanceStat
 
         return retVal;
     }
+
+    @Override
+    public RepoGenericResponse<Elastigroup> getBeanstalkConfig(String environmentId, String region, String authToken, String account) {
```

Comment:
re-arrange the functions as is in openApi document

Reply:
Done